run.py

```diff
@@ -31,8 +31,8 @@
 try:
     from webcompat import app
 except ImportError, e:
-    if e.import_name == 'config':
-        # config not found, somebody forgot to make a copy of config.py.example and modify?
+    if 'import_name' in e and e.import_name == 'config':
+        # config not found, did you forget to copy config.py.example?
```

Comment:
nitpick: I was about to fix this in another commit. Here the comment says what the dev should do (which is the error message), instead of documenting the code. ^_^ I would put instead. 

```python
# Catching the missing config.py file
```
The `NO_CONF_ERROR` explains what the dev has to do.